avro-fastserde/src/jmh/java/com/linkedin/avro/fastserde/FastAvroSerdesBenchmark.java

```diff
@@ -126,22 +127,48 @@ public void testFastAvroSerialization(Blackhole bh) throws Exception {
   @Benchmark
   @OperationsPerInvocation(NUMBER_OF_OPERATIONS)
   public void testAvroDeserialization(Blackhole bh) throws Exception {
-    testDeserialization(deserializer, bh);
+    testDeserialization(deserializer, bh, 0);
   }
 
   @Benchmark
   @OperationsPerInvocation(NUMBER_OF_OPERATIONS)
   public void testFastAvroDeserialization(Blackhole bh) throws Exception {
-    testDeserialization(fastDeserializer, bh);
+    testDeserialization(fastDeserializer, bh, 0);
   }
 
-  private void testDeserialization(DatumReader<GenericRecord> datumReader, Blackhole bh) throws Exception {
+  @Benchmark
+  @OperationsPerInvocation(NUMBER_OF_OPERATIONS)
+  public void testFastAvroDeserializationAccess(Blackhole bh) throws Exception {
+    testDeserialization(fastDeserializer, bh, 1);
+  }
+
+  @Benchmark
+  @OperationsPerInvocation(NUMBER_OF_OPERATIONS)
+  public void testFastAvroDeserializationAccess8(Blackhole bh) throws Exception {
+    testDeserialization(fastDeserializer, bh, 8);
+  }
```

Comment:
The new benchmark tests and the following changes in `testDeserialization()` are just to benchmark FloatArray SerDes. It may break the JMH benchmark if the BenchmarkSchema is changed... I would suggest not to commit these changes. Or, you can create a new Benchmark just for this specific case.

Reply:
Sure, will make a new benchmark for this case.